frontend/src/components/common/comments-list/comments-list.tsx

```diff
@@ -18,22 +19,28 @@ const CommentsList: React.FC<Props> = ({
   onTimeClick,
   hasTimestamps,
   onCommentDelete,
+  onCommentLike,
 }) => {
 
   const isMaster = user?.role === UserRole.MASTER;
 
   return (
     <ul className={styles.list}>
       {comments.map((item) => {
+        const isOwner = user?.id === item.userId;
         const isAllowDelete = user?.id === item.userId || isMaster;
+        const isLiked = item.commentReactions?.some((reaction) => reaction.userId === user?.id);
```

Comment:
create `checkIsLiked` helper and move it to `frontend/src/components/common/comments-list/helpers/`